packages/node_modules/@webex/plugin-meetings/test/unit/spec/index.js

```diff
@@ -0,0 +1,260 @@
+
+import {assert} from '@ciscospark/test-helper-chai';
```

Comment:
didn't look deeply, but looks like this is a duplicate from device-manager?

Reply:
Not sure about this